frontend/src/components/common/StudentDetailCommon.vue

```diff
@@ -574,6 +588,28 @@ export default {
     formatPen,
     formatMincode,
     formatDob,
+    boldFormatter: function (char){
```

Comment:
Could you please use ES6 syntax to define methods? like `boldFormatter(char) { .. }`, same suggestion for other added methods.